bzkanban.js

```diff
@@ -1062,32 +1079,48 @@ function dropCard(ev) {
     }
 }
 
-function dropBacklog(ev) {
-    if (ev.target.classList.contains("drop-target")) {
-        ev.target.classList.remove("drop-target-hover");
-    }
-    ev.preventDefault();
+function showBacklog() {
+    var button = document.getElementById("btnShowBacklog");
+    var backlogCol = document.querySelector("#BACKLOG.board-column");
+
+    if (backlogCol.style.display === "none" || null) {
+        // Load backlog on first access.
+        var backlog = backlogCol.querySelector(".cards");
+        if (backlog.children.length == 0) {
+            loadBacklogCards();
```

Comment:
Ya not sure we should load the backlog cards in `loadBoard`. Some backlogs are pretty massive. And unless the user opens the backlog, which is probably a somewhat rare action, it'll be an unnecessary extra load happening.